aviary/docs/user_guide/aviary_commands.ipynb

```diff
@@ -420,10 +460,11 @@
     "```\n",
     "`aviary convert_prop_table -f GASP PropFan.map PropFan.prop` Convert a GASP based propeller map\n",
     "`aviary convert_prop_table -f GASP general_aviation.map general_aviation.prop` Convert a GASP based propeller map\n",
+    "`aviary convert_prop_table general_aviation.map` Convert a GASP based propeller map\n",
     "```\n",
     "\n",
     "The first example uses Mach number and the second example uses helical Mach number. \n",
-    "Note that the output file name can be skipped. Since there is only one input data format is supported at this time, input data format is defaulted to GASP if not provided."
+    "Note that the output file name can be skipped as demonstrated in the third example. Since there is only one input data format is supported at this time, input data format is defaulted to GASP if not provided. This is shown in the third example as well."
```

Comment:
```suggestion
    "Note that the output file name can be skipped as demonstrated in the third example. Since there is only one input data format that is supported at this time, it defaults to GASP if not provided. This is shown in the third example as well."
```